test/spec/unit/configparser.js

```diff
@@ -9,7 +9,8 @@ describe('ConfigParser', () => {
         configParser.addConfigFile(FIXTURES_PATH + '/exclude.conf.js')
         let specs = configParser.getSpecs()
         specs.should.not.include(__filename)
-        specs.should.include(__dirname + '/call.js')
+        specs.should.include(path.resolve(__dirname + '/call.js'))
+        specs.should.include(path.resolve(__dirname + '/typescript.ts'))
```

Comment:
Could we move this file into the fixtures directory and modify the exclude.conf.js to also point to lets say all ts files in that fixture directory?